app/views/support/WeeklyPromotion.scala

```diff
@@ -7,63 +7,57 @@ import com.gu.memsub.promo.{PromoCode, WeeklyLandingPage}
 import com.gu.memsub.subsv2.{CatalogPlan, Catalog => SubsCatalog}
 import com.netaporter.uri.Uri
 import com.netaporter.uri.dsl._
+import model.GuardianWeeklyZones
 import views.support.Pricing._
 
 object WeeklyPromotion {
   private val UK = CountryGroup.UK.countries.toSet
   private val UKdomestic = Set(CountryGroup.C.UK)
   private val UKoverseas = UK diff UKdomestic
-  private val US = CountryGroup.US.countries.toSet
-  private val AU = CountryGroup.Australia.countries.toSet
-  private val NZ = CountryGroup.NewZealand.countries.toSet
-  private val EU = CountryGroup.Europe.countries.toSet
-  private val CA = CountryGroup.Canada.countries.toSet
+
   private val allCountries = CountryGroup.allGroups.flatMap(_.countries).toSet
-  private val ZONEC = allCountries.diff(UK union US)
+
   case class DiscountedPlan(currency: Currency, pretty: String, headline: String, period: String, url: Uri, discounted: Boolean)
 
-  case class DiscountedRegion(title: String, description: String, countries: Set[Country], discountedPlans: List[DiscountedPlan])
+  case class DiscountedRegion(title: String, description: String, discountedPlans: List[DiscountedPlan])
 
   def validRegionsForPromotion(promotion: Option[PromoWithWeeklyLandingPage], promoCode: Option[PromoCode], requestCountry: Country)(implicit catalog: SubsCatalog): Seq[DiscountedRegion] = {
     val promotionCountries = promotion.map(_.appliesTo.countries).getOrElse(allCountries)
-    val regionForZoneCCountry: Seq[DiscountedRegion] = {
-      if (ZONEC.contains(requestCountry)) {
+
+    val separateRegionIfRequestCountryIsRestOfWorld: Seq[DiscountedRegion] = {
```

Comment:
I would call this 'promotedRegion' and add a comment with something like:

// If a user does not qualify for domestic delivery (e.g. if the user is based in South Africa), we want to explicitly call out their (likely) delivery country on the landing page

Or preferably a simpler way to phrase that ^^^ 🙈

Reply:
Yeah, good shout. Names are hard. 😬